src/emit/solana.rs

```diff
@@ -3602,21 +3602,18 @@ impl<'a> TargetRuntime<'a> for SolanaTarget {
         function: FunctionValue<'b>,
         _event_no: usize,
```

Comment:
I can exchange them by an underscore. I cannot remove them because the trait dictates the arguments. Keep in mind that there is a separate implementation of this for Substrate.